appservice/src/SiteWrapper.ts

```diff
@@ -65,15 +69,21 @@ export class SiteWrapper {
             await client.webApps.listPublishingCredentials(this.resourceGroup, this.name);
     }
 
-    public async getSiteConfig(client: WebSiteManagementClient): Promise<SiteConfig> {
+    public async getSiteConfig(client: WebSiteManagementClient): Promise<SiteConfigResource> {
         return this.slotName ?
             await client.webApps.getConfigurationSlot(this.resourceGroup, this.name, this.slotName) :
             await client.webApps.getConfiguration(this.resourceGroup, this.name);
     }
 
+    public async updateConfiguration(client: WebSiteManagementClient, config: SiteConfigResource): Promise<SiteConfigResource> {
+        return this.slotName ?
+            await client.webApps.updateConfigurationSlot(this.resourceGroup, this.appName, config, this.slotName) :
+            await client.webApps.updateConfiguration(this.resourceGroup, this.appName, config);
+    }
+
     public async deployZip(fsPath: string, client: WebSiteManagementClient, outputChannel: vscode.OutputChannel): Promise<void> {
         const yes: string = 'Yes';
-        const warning: string = `Are you sure you want to deploy to "${this.appName}"? This will overwrite any previous deployment and cannot be undone.`;
+        const warning: string = localize('zipWarning', `Are you sure you want to deploy to "${this.appName}"? This will overwrite any previous deployment and cannot be undone.`);
```

Comment:
For strings like this you want to insert values slightly differently (It can't be translated the current way you have it):
```
localize('zipWarning', 'Are you sure you want to deploy to "{0}"? This will overwrite any previous deployment and cannot be undone.', this.appName);
```